### ISSUE TITLE
JS minify failure

### ISSUE THREAD

**Issue by jmooring** [tdewolff/minify#449]
```js
const _\u00bare_unicode_escape_identifier = 42;
console.log(_\u00bare_unicode_escape_identifier);
```

```
unexpected \u00bare_unicode_escape_identifier in const declaration on line 1 and column 8
    1: const _\u00bare_unicode_escape_identifier = 42;
              ^
```

Reference: <https://github.com/gohugoio/hugo/issues/9244>

**Comment by tdewolff** [comment]
Thanks for the bug report! This bug was recently fixed with https://github.com/tdewolff/parse/commit/3983db1b37eeb8922a71c25e55db2dbafb09633f and is in the `master` branch here. I'm waiting to bring out a new release due to a large commit for variable renaming and reordering which needs further testing first. Fix will be released ASAP

**Comment by tdewolff** [comment]
Fixed some bugs in 2cb24efe8ed0469a7fdc669a5d0ba5bef62658b4 and https://github.com/tdewolff/parse/commit/eca56c88263ed84c15620669a64e38f067213f3d after testing. Released fix in https://github.com/tdewolff/minify/releases/tag/v2.9.23, please report any bugs!
